### PR TITLE
Issue #DP-541 Task: In textbook ToC page, ToC rows should display the total number of contents under it

### PR DESCRIPTION
**Pull request by niteshkesarkar**
Currently, there are some console logs or some debug code is present. Please ignore these now and check for the implementation. Will remove this debug code before I ask for merge.

### CODE REVIEW

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts [hunk 1]
```diff
@@ -736,12 +737,27 @@ export class ChapterListComponent implements OnInit, OnChanges, OnDestroy, After
 
 
   filterContentsForCount (contents, status?, onlySample?, organisationId?, createdBy?) {
+    if (organisationId) {
+      organisationId = '1-' + organisationId;
+    }
     const filter = {
       ...(status && {status}),
       ...(onlySample && {sampleContent: true}),
+      ...(!onlySample && {sampleContent: null}),
```
Comment (venkateshwarans) [inline src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts:746]: You made sure the `_.filter` is working when `{sampleContent: null}` ?
And when we want only `sampleContent`s count, it is working? (For sourcing org admin)

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts [hunk 2]
```diff
@@ -736,12 +737,27 @@ export class ChapterListComponent implements OnInit, OnChanges, OnDestroy, After
...
+      filter.createdBy = this.currentUserID;
+    }
+    if (this.isContributingOrgReviewer()) {
+      filter.status = 'Review';
+    }
+    if (this.isSourcingOrgReviewer()) {
+      filter['visibility'] = true;
+    }
+    console.log('filter', filter);
+    _.map(contents, leaf => console.log(leaf.name + ' == ' + leaf.status + ' == ' + leaf.createdBy + ' == ' + leaf.sampleContent + ' == ' + leaf.organisationId));
```
Comment (venkateshwarans) [inline src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts:760]: Unless you have kept it for debugging on dev, remove console

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts (hunk 1 above)
Comment (niteshkesarkar) [reply src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts:746]: `For sourcing org admin` - I am yet to test, as a hierarchy API sync issue is not yet resolved. Once that done will test for the sourcing side. Yes, for contribution side it's working with `sampleContent: null`

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts (hunk 2 above)
Comment (niteshkesarkar) [reply src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts:760]: this is for only testing on local will remove before asking to merge.

File: src/app/client/src/app/modules/cbse-program/services/collection-hierarchy/collection-hierarchy.service.ts [hunk 3]
```diff
@@ -260,7 +260,7 @@ export class CollectionHierarchyService {
 
     if (!_.isUndefined(sampleContentCheck)) {
       option.data.request.filters['sampleContent'] = sampleContentCheck;
-      option.data.request.filters['status'] = ['Draft', 'Review'];
```
Comment (pallakartheekreddy) [inline src/app/client/src/app/modules/cbse-program/services/collection-hierarchy/collection-hierarchy.service.ts:263]: WHy you removed `Draft`?

**Pull request merged**